src/DdSdkReactNative.tsx

```diff
@@ -31,6 +34,8 @@ export class DdSdkReactNative {
                 return
             }
 
+            configuration.additionalConfig[this.DD_SOURCE_KEY] = 'react-native';
```

Comment:
The problem with this solution is that it only works if people initialize the SDK using the `DdSdkReactNative` class. If they use the `DdSdk` class instead, the source won't be changed. i'd rather hardcode it in both the iOS and Android side of the sdk instead.

Reply:
Do we want to expose `DdSdk`? It may be a bit confusing for the end user to have 2 SDK-related classes: `DdSdk` and `DdSdkReactNative`.

Reply:
That was actually my thought also, just to hardcoded in the `Android/iOS` code. Also regarding the 2 initialization classes I do agree with @0xnm and probably we should only expose the `DdSdkReactNative` class.

Reply:
I pushed the update where the source is implemented via native classes.

Reply:
Rolled back to the original TS implementation.